Ansible/roles/kubernetes/tasks/kubernetes.yml

```
@@ -7,7 +7,9 @@
     baseurl: https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
     gpgcheck: true
     repo_gpgcheck: true
-    gpgkey: https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+    gpgkey: >
+      https://packages.cloud.google.com/yum/doc/
+      yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
```

It might make more sense for one URL per line here

```suggestion
      https://packages.cloud.google.com/yum/doc/yum-key.gpg
      https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
```